### PR TITLE
P4TR: add column about backups and security schemes

### PR DESCRIPTION
**Pull request by harding**
Tentatively scheduled for Sept 15th.  Early reviews welcome!

### CODE REVIEW

File: _includes/specials/taproot/en/xx-backups.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+- **Compromise detection:** an idea [proposed][tree signatures] prior to
+  the invention of taproot is to put a key controlling some amount of
+  bitcoin on all of the devices you care about as a way of detecting
+  when the device has been compromised.  If the amount of bitcoin is
+  large enough, the attacker will probably spend it to themselves for
+  the immediate gain rather than wait to use their illicit access in a
+  long-term attack that might cause you more overall harm.
+
+    A problem with this approach is that you want to make the amount of
```
Comment (bitschmidty) [inline _includes/specials/taproot/en/xx-backups.md:85]: Cool idea!

**Review by bitschmidty — COMMENTED** [review]
LGTM utACK

File: _includes/specials/taproot/en/xx-backups.md (hunk 1 above)
Comment (harding) [reply _includes/specials/taproot/en/xx-backups.md:85]: Exactly what I thought when I first read it!  :-D

**Comment by harding** [comment]
Closing this PR now.  Will be moving the content into a newsletter PR, hopefully to be opened later today.  Thanks so much for the early reviews @Xekyo and @bitschmidty!

**Pull request closed without merging**